docs/wiki/quick_start.md

```diff
@@ -117,46 +138,37 @@ To start the wizard click the left menu on Solutions then Network then Create ne
 
 ![solutions menu](images/solutions_list.png)
 
-First step is to choose the name of your network. This name is only used to identify the network later on when deploying workloads.
-
-![choose network name](images/network_name.png)
-
-Then choose the currency that will be used for paying for all the deployments on your network
-
-![choose network currency](images/network_currency.png)
+First step is to choose you want to create new network or add access to specific node in your network.
```

Comment:
`choose whether you want to create a new network or add access to a specific node...`